src/components/ActivityRibbon.vue

```diff
@@ -12,6 +12,9 @@
         {{ tab.text }}
       </span>
     </FilterButton>
+    <div class="right">
+      <slot />
+    </div>
```

Comment:
instead of nesting flexboxes, you can add a node stretched between the left and right sides
```suggestion
    <div class="separator" style="flex-grow: 1" />
    <slot />
```